cpp/velox/utils/VeloxArrowUtils.cc

```diff
@@ -58,7 +58,7 @@ arrow::Status MyMemoryPool::Allocate(int64_t size, int64_t alignment, uint8_t**
   if (bytes_allocated() + size > capacity_) {
     return arrow::Status::OutOfMemory("malloc of size ", size, " failed");
   }
-  RETURN_NOT_OK(pool_->Allocate(size, alignment, out));
+  RETURN_NOT_OK(pool_->Allocate(size, out));
```

Comment:
Is this change needed?

Reply:
Yes, because arrow version is downgraded, and the API changed.

Reply:
The arrow version is upgraded from arrow 12.0 to 13.0, so I assume alignment could be kept.